### ISSUE TITLE
request: qch support

### ISSUE THREAD

**Issue by RJVB** [zealdocs/zeal#690]
Since Zeal is written in Qt it might not be particularly hard to add support for Qt's own compiled help (.qch) files? That would make Zeal an integrated solution for 2 readily available documentation bundle formats, one of which almost everyone who builds Zeal already has a collection of.

I for one would love to move from Qt's assistant to a browser that also has access to Mac SDK documentation, and that doesn't discard my whole documentation collection every time I update Qt.

**Comment by trollixx** [comment]
This is interesting, and I [mentioned](https://github.com/zealdocs/zeal/issues/285#issuecomment-82684428) this before, but we need a pluggable docset format architecture.

**Comment by RJVB** [comment]
Yes, though I'd advise against actual plugins if you're not planning to encourage 3rd-party contributions.
